dhis-2/dhis-services/dhis-service-core/src/main/resources/i18n_global.properties

```diff
@@ -428,6 +428,8 @@ F_UNCOMPLETE_EVENT=Uncomplete events
 F_SKIP_DATA_IMPORT_AUDIT=Skip data import audit
 F_TRACKER_IMPORTER_EXPERIMENTAL=Tracker importer experimental
 F_VIEW_SERVER_INFO=View server information
+F_DATA_APPROVAL_WORKFLOW=Add/Update/Delete Data Approval Workflow
```

Comment:
I think the translation should be just `Add/Update Data Approval Workflow` to be consistent with the others @vietnguyen ?

Reply:
fixed